apps/voting/contracts/Voting.sol

```diff
@@ -235,6 +257,16 @@ contract Voting is IForwarder, AragonApp {
         return _canVote(_voteId, _voter);
     }
 
+    /**
+    * @notice Tells whether `_sender` can object to a vote #`_voteId` by voting 'no'
+    * @dev Initialization check is implicitly provided by `voteExists()` as new votes can only be
+    *      created via `newVote(),` which requires initialization
+    * @return True if the given voter can participate a certain vote, false otherwise
```

Comment:
I suggest renaming it from 'can participate' to 'can object' or 'vote against?

Reply:
@return True if the given voter can participate in the objection phase of a certain vote, false otherwise